vertical-pod-autoscaler/pkg/recommender/model/vpa.go

```diff
@@ -164,3 +170,42 @@ func (vpa *Vpa) matchesAggregation(aggregationKey AggregateStateKey) bool {
 	}
 	return vpa.PodSelector != nil && vpa.PodSelector.Matches(aggregationKey.Labels())
 }
+
+// AsStatus returns this objects equivalent of VPA Status.
+func (vpa *Vpa) AsStatus() *vpa_types.VerticalPodAutoscalerStatus {
```

Comment:
Do I understand correctly that this function requires UpdateConditions being called first? If so - please comment as such.
The same applies to HasMatchedPods.

Reply:
Done.